politeiawww/piwww.go

```diff
@@ -243,6 +245,36 @@ func convertCensorshipRecordFromPD(cr pd.CensorshipRecord) pi.CensorshipRecord {
 	}
 }
 
+func convertCommentsPluginPropStateFromPi(s pi.PropStateT) comments.StateT {
+	switch s {
+	case pi.PropStateUnvetted:
+		return comments.StateUnvetted
+	case pi.PropStateVetted:
+		return comments.StateVetted
+	}
+	return comments.StateInvalid
+}
+
+func convertPluginCommentToPi(cm comments.Comment, state pi.PropStateT, username string) pi.Comment {
```

Comment:
Keep with the current conversion naming convetion.

`convert [typeBeingReturned] From [package]`

For this it would be `convertCommentFromComments`.  That sound a little weird so something like this is also fine `convertCommentFromPlugin`.  It conveys the same information.

You don't need to pass in the state argument.  It's already in a `comments.Comment`.  You just need to convert it.

For the sake on consistency, don't fill in the user information if its not included in the `comments.Comment`.  Convert strictly from one type to another type.  See the proposal record conversion function as an example.  Fill in the user information in the calling function.

https://github.com/lukebp/politeia/blob/tlog/politeiawww/piwww.go#L309-L329

Reply:
makes sense, will do